Sources/ToastCenter.swift

```diff
@@ -18,6 +18,8 @@ open class ToastCenter: NSObject {
   /// VoiceOver will announce the text in the toast when `ToastView` is displayed.
   @objc public var isSupportAccessibility: Bool = true
   
+  @objc public var isQueueEnabled: Bool = true
```

Comment:
Please add a `///` documentation comment above this new declaration, thank you.

Reply:
I added the documentation in 9e0fb9c83eb411175b09533d3e09848e7888859d